documentation/book/con-interbroker-communications.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: assembly-cluster-operator.adoc
+// * ID: [id='encryption-{context}']
+// * Title: = Encryption
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+
+include::common/attributes.adoc[]
+
+[id='interbroker-communication-{context}']
+= Interbroker Kafka communication
+
+Communication between brokers is done through the `REPLICATION` listener on port 9091 and is encrypted by default.
```

Comment:
Is there anymore to this file?

Reply:
I am not sure. This is a low priority user story and can be revisited after the beta release.

Reply:
Then we shouldn't bother including it, doing it wrong is worse than not having it.